libcxx/test/configs/ve-libc++-shared.cfg.in

```diff
@@ -0,0 +1,28 @@
+# This testing configuration handles running the test suite against LLVM's
```

Comment:
This should come with changes to `libcxx/utils/ci/run-buildbot` and `libcxx/utils/ci/buildkite-pipeline.yml` that allow running these tests in pre-commit. You can reach out to me on Discord privately and I'll help you with the BuildKite setup (agent tokens and so on).

Reply:
Thanks.  I think I can use a machine runnning a buildbot for both purpose, buildbot and buildkite.  I'm discussing internally.